syspage.h

```diff
@@ -100,6 +100,9 @@ extern void syspage_addEntries(addr_t start, size_t sz);
 extern int syspage_getMapAttr(const char *map, unsigned int *attr);
 
 
+extern char *syspage_getMapName(u32 id);
```

Comment:
What do You think about returning `const char *`?

Reply:
Done.